docs/CHILI-GraFx/guides/setup-fsso/index.md

```diff
@@ -13,7 +13,7 @@ Once the CHILI Publish team configures the connection, regardless of the preferr
 To verify this, please follow this process:
 
 1. Open a fresh browser via Private or Incognito browsing
-2. Go to http://chiligrafx.com
+2. Navigate to [GraFx](https://chiligrafx.com)
```

Comment:
very nitpicky - Navigate is quite "complex" word - it implies that it will be "hard/complex/requite lot of steps" 
I feel like "Go to" is simpler/easier

Reply:
Replaced it